go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/giantswarm/k8scloudconfig
+
+go 1.13
+
+require (
+	github.com/Masterminds/semver v1.5.0
+	github.com/giantswarm/apiextensions v0.2.0
+	github.com/giantswarm/microerror v0.2.0
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/stretchr/testify v1.4.0
+	golang.org/x/net v0.0.0-20191207000613-e7e4b65ae663 // indirect
```

Comment:
FYI you can usually remove these indirects and run `go mod tidy` again. It is usually fine. I encountered it sometimes adds some of them back and then it stays but that way we can try to keep it a bit more tidy.